sass/_z-index.scss

```diff
@@ -5,16 +5,19 @@
 // Z-Index
 // -------
 /// A list of named z-index layers, from lowest to highest index.
-/// Nest lists (one level deep) for different z-index contexts.
+/// Nest lists (one level only) to establish a new z-index context:
+/// The first item will be indexed based on top-level list position,
+/// while additional values are indexed
+/// by their position in the nested list.
 /// @type List
 /// @group z-index
 /// @example scss -
 ///   $z-index: (
-///     'main',
-///     'sidebar',
-///     'banner' 'active-nav' 'dropdown',
-///     'message',
-///     'modal',
+///     'main', // 1
+///     'sidebar', // 2
+///     'banner' 'active-nav' 'dropdown', // 3 2 3
```

Comment:
is there not a 1 if it is nested? I would have guess it would be either `3 1 2 ... ` or  `3 4 5 ...`

Reply:
Good thinking. `3 4 5…` is the same as not nesting, so probably not the way to go, but `3 1 2…` makes sense to me. I'll make that change.